src/test/java/com/sfuentes/OrangeShopTest.java

```diff
@@ -0,0 +1,87 @@
+package com.sfuentes;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class OrangeShopTest {
+
+  @Test
+  void calculateFifteenPercentage_integerNumber() {
+    OrangeShop client = new OrangeShop();
+    int number = 100;
+    double expectedResult = 15;
+
+    assertEquals(expectedResult, client.calculateFifteenPercentage(number));
+  }
+
+  @Test
+  void calculateFifteenPercentage_decimalNumber() {
+    OrangeShop client = new OrangeShop();
+    double number = 500.50;
+    double expectedResult = 75.075;
+
+    assertEquals(expectedResult, client.calculateFifteenPercentage(number));
+  }
+
+  @Test
+  void calculateCustomerPayment_tenKilos() {
```

Comment:
I think there is one scenario left for the method calculateCustomerPayment: when is less than 10 kilos

Reply:
Resolved